portfolio/src/main/webapp/index.html

```diff
@@ -4,17 +4,29 @@
     <meta charset="UTF-8">
     <title>Andrew Pham | Home</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <script src="script.js">
+        
+    </script>
   </head>
   <body>
     <div id="content">
       <h1> Andrew Portfolio</h1>
       <h3>Hello! My name is Andrew Pham and I am a second year at the University of California San Diego. I grew up with a family of six: two parents, an older brother and sister, <br>
-            and a younger brother. My favorite color is purple and the type of music I like to listen to is Kchill and lofi.
+            and a younger brother. My favorite color is purple and the type of music I like to listen to is Kchill and lofi.<h3>
       <p>This is my portfolio.</p>
+    </div>
+
+      <div id="left">
+        <img src="/images/me.jpg" width="100%">
```

Comment:
Scott suggested to me that for images anything like width or height be put in a css file (in this case the one referenced above "style.css")